timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/server/endpoints/v2/Gremlin.java

```diff
@@ -293,12 +300,19 @@ private void dumpVertex(Vertex vertex, StringBuilder result) {
   }
 
   private void dumpEdge(Edge edge, StringBuilder result) {
+    ObjectMapper map = new ObjectMapper();
```

Comment:
Objectmapper kan een field worden en worden hergebruikt in zowel dumpVertex als dumpEdge.